src/01.UI-Elements/InputForm.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+import AddIcon from '@material-ui/icons/Add'
+
+InputForm.propTypes = {
+  onCreateListItem: PropTypes.func.isRequired,
+}
+
+export default function InputForm({ onCreateListItem }) {
+  return (
+    <form onSubmit={handleSubmit}>
+      <Element>
+        <AddIconStyled />
+        <InputFieldStyled
+          name="title"
+          type="text"
+          placeholder="Listeneintrag"
+        />
+      </Element>
+    </form>
+  )
+  function handleSubmit(event) {
+    event.preventDefault()
+    const formElement = event.target
+    const input = formElement.elements.title
+    onCreateListItem(input.value)
+    formElement.reset()
+  }
+}
+
+const Element = styled.label`
```

Comment:
For me the name "Element" is not explicit enough.

Reply:
Renamed it. thx